libs/test/SalesforceSDKTest/src/com/salesforce/androidsdk/rest/files/FileRequestsTest.java

```diff
@@ -185,18 +183,16 @@ public void testFileShares() {
      * @throws Exception
      */
     public void testFileUpload() throws Exception {
-        final String filename  = "MyFile.txt";
```

Comment:
We were getting a permission denied on Android O.
We don't actually need to create a file on external storage, so using File.createTempFile instead.